src/index.ts

```diff
@@ -201,7 +201,13 @@ extendEnvironment((hre) => {
 
 task("starknet-verify", "Verifies the contract in the Starknet network.")
     .addOptionalParam("starknetNetwork", "The network version to be used (e.g. alpha)")
-    .addParam("path", "The path of the cairo contract (e.g. contracts/conract.cairo)")
+    .addOptionalParam("path", "The path of the cairo contract (e.g. contracts/conract.cairo)")
+    .addOptionalParam(
```

Comment:
Why have both `path` and `paths`? Shouldn't it be just `path` which is a variadic parameter? Also that way users wouldn't have to quote their space separated paths.

Another approach could be to keep the `path` parameter and introduce a `dependencies` optional variadic parameter.